src/pudl/transform/classes.py

```diff
@@ -434,6 +480,54 @@ def rename_columns(
             )
         return df.rename(columns=params.columns)
 
+    def remove_invalid_rows(
+        self, df: pd.DataFrame, params: RemoveInvalidRows
+    ) -> pd.DataFrame:
+        """Drop rows with only invalid values in all specificed columns.
+
+        This method finds all rows in a dataframe that contain ONLY invalid data in ALL
+        of the columns that we are checking and drops them with a notification of the %
+        of the records that were dropped. The input parameters must specific the invlaid
+        values to search for and only one of the following:
+        * columns to seach as ``cols_to_check``. This will be used directly in
+          ``pandas.filter(items)``
+        * columns to not search as ``cols_to_not_check``. This is usefull if a table
+          is large and specifcying all of the ``cols_to_check`` would be arduous.
+        * a string to search on via ``pandas.filter(like)`` as ``like``
+        * a regex to search on via ``pandas.filter(regex)`` as ``regex``
```

Comment:
I wasn't sure if this is the right place for this information. It felt better to have it here over `RemoveInvalidRows` because this is more of the interface for the user... but idkl

Reply:
I think we need to document the attributes of all the `TransformParam` models, since that's where their interface is defined. Sorry I wasn't good about doing that in the initial draft though. 😦